utilities/installer/SAMPLE_host.cfg

```diff
@@ -15,11 +15,23 @@
 # Replace the < > with your information
 
 HOSTS:
-    <agent1>:
+    <IOS XR node fully qualified domain name>:
```

Comment:
This is actually the name (FQDN) that the agent identifies itself by - it's what gets populated into puppet.conf (`certname = xr-agent.domain.com`). There's no DNS lookup involved - my XRv9k node is not in DNS but this works fine.